index.js

```diff
@@ -28,6 +28,9 @@ module.exports = class Trailpack {
     if (!pack.config) {
       pack.config = { }
     }
+    if (!pack.config.trailpack) {
```

Comment:
And here https://github.com/catrielmuller/trailpack/blob/50d3d13cb4b03fc0f5cd56e5c303fb5faf15fe20/index.js#L45 this empry object recive the default configurations for the trailpack. 
I think we dont need a error message if the Trailpack use the base config if dont exist